e2etests/tests/graceful_restart.go

```diff
@@ -133,3 +157,27 @@ var _ = ginkgo.Describe("Establish BGP session with EnableGracefulRestart", func
 		)
 	})
 })
+
+// checkPrefixOnNode checks that prefixes have at one kernel route on the
+// the node. This function get routes directly from the node `docker exec ip
+// route` and not from `vtysh show route` because we need to get routes while
+// FRR process restarts.
+func checkPrefixOnNode(node string, prefixes []string) error {
+	exc := executor.ForContainer(node)
```

Comment:
sorry I just noticed this. If we do this, this test won't be able to run if/when we run on a real cluster. I think what other tests are doing (or, I hope) is to check on the frr-k8s pods. Should be equivalent as they are host - routed.

Reply:
good catch, but frr-k8s pods are restarting, so that is not an option.
In real cluster that would have to be ssh or find another hsot-routed pod (or create one)

Reply:
Fair enough, no need to happen here.